mods/screwdriver/init.lua

```diff
@@ -80,6 +130,7 @@ screwdriver.handler = function(itemstack, user, pointed_thing, mode, uses)
 	if should_rotate then
 		node.param2 = new_param2
 		minetest.swap_node(pos, node)
+		minetest.check_for_falling(pos)
```

Comment:
Imagine that you want to turn a sign 180 degrees with the screwdriver (the sign is between two nodes). You click once - then the sign pops down, since it only turned 90 degrees!

This is why the first falling check is necessary - it detects if a place is not suitable and skips that orientation.